label_studio/projects/serializers.py

```diff
@@ -82,6 +98,46 @@ def user_id(self):
     def get_config_has_control_tags(project):
         return len(project.get_parsed_config()) > 0
 
+    @staticmethod
+    def get_config_suitable_for_bulk_annotation(project):
+        li = LabelInterface(project.label_config)
+
+        # List of tags that should not be present
+        disallowed_tags = [
+            LabelsTag,
+            EllipseLabelsTag,
+            KeyPointLabelsTag,
+            PolygonLabelsTag,
+            RectangleLabelsTag,
+            HyperTextLabelsTag,
+            ParagraphLabelsTag,
+            TimeSeriesLabelsTag,
```

Comment:
`BrushLabels` are missing + all usual image region tags should also be a **no**: `Brush`, `Ellipse`, `KeyPoint`, `Polygon`, `Rectangle`
```suggestion
            LabelsTag,
            EllipseLabelsTag,
            KeyPointLabelsTag,
            PolygonLabelsTag,
            RectangleLabelsTag,
            HyperTextLabelsTag,
            ParagraphLabelsTag,
            TimeSeriesLabelsTag,
```

Reply:
Plus maybe `MagicWand`

Reply:
@hlomzik Should we do smthng with `VideoRectangle`, `TimelineLabels`?

Reply:
Oh, yes, thank you! All these 3 tags should also be banned =)

Reply:
I added VideoRectangle, unfortunately, TimelineLabels and MagicWand is not supported by our parsing library yet, we can add them in a follow-up ticket.